Resources/Prototypes/_DV/Body/Organs/oni.yml

```
@@ -0,0 +1,33 @@
+- type: entity
+  id: OrganOniStomach
+  parent: OrganHumanStomach
+  name: oni stomach
+  description: Ew, this is hard to stomach. Larger than normal
+  components:
+  - type: Stomach
+    specialDigestible:
+      tags:
+      - ReptilianFood
+      - Meat
+      - Pill
+      - Crayon
+      - Paper
+  - type: SolutionContainerManager
+    solutions:
+      stomach:
+        maxVol: 80 # Bigger Body = Bigger Stomach
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: UncookedAnimalProteins
+          Quantity: 5
+  - type: Metabolizer
+    metabolizerTypes: [ Oni, Animal ]
```

Remove spaces for consistency
```suggestion
    metabolizerTypes: [Oni, Animal]
```